zanata-war/src/main/java/org/zanata/security/ZanataOpenId.java

```diff
@@ -339,33 +344,28 @@ public String getRedirectToUrl() {
         return null;
     }
 
-    public void setProvider(OpenIdProviderType providerType) {
-        if (providerType != null) {
-            switch (providerType) {
+    @Produces
+    @Unambiguous
```

Comment:
Ahh, is that the purpose of that annotation?
Because yes, the purpose is just to make it unambiguous.